dist/index.js

```diff
@@ -184,8 +197,13 @@ async function downloadAsset(client, owner, repo, releaseAsset, directory) {
             throw 'Not Found'
         }
 
-        file.write(Buffer.from(response.rawBody));
-        file.end();
+        // I'm now no longer 100% sure this was the culprit, lets log just in case it bites us again
```

Comment:
maybe rethink the wording of this comment, since the reader won't have the context of this bug fix. I actually don't think you need a comment at all. It's good to log things, no need to explain.

Reply:
fair!